### PR TITLE
Update SVM The Escape.json

### PR DESCRIPTION
**Pull request by devilflippy**
Remove some config, trying to fixe incompatibility with Realism/Donuts, and some other mods
Fence prices on x10 to see if SVM take priority

### CODE REVIEW

File: user/mods/ServerValueModifier/Presets/The Escape.json [hunk 1]
```diff
@@ -116,14 +116,14 @@
...
-    "RemoveTimeCondition": false,
+    "RemoveTimeCondition": true,
     "AllQuestsAvailable": false,
     "RemoveBarterOffers": false,
     "RemoveCurrencyOffers": false,
     "IncreaseAssort": false,
     "UnlockQuestAssort": false,
     "EnableTraders": true,
-    "FIRRestrictsQuests": false,
+    "FIRRestrictsQuests": true,
```
Comment (guillaumearm) [inline user/mods/ServerValueModifier/Presets/The Escape.json:126]: This is already handled by Realism

File: user/mods/ServerValueModifier/Presets/The Escape.json [hunk 2]
```diff
@@ -499,22 +499,22 @@
...
       "FourthH": 1,
       "FourthV": 1,
-      "ThirdH": 1,
-      "ThirdV": 2,
-      "SecondH": 1,
-      "SecondV": 2,
+      "ThirdH": 2,
+      "ThirdV": 1,
+      "SecondH": 2,
+      "SecondV": 1,
```
Comment (guillaumearm) [inline user/mods/ServerValueModifier/Presets/The Escape.json]: Even if it's disabled, this is not the correct values here

**Review by guillaumearm — CHANGES_REQUESTED** [review]
Some changes are needed, after that we're good to go.

File: user/mods/ServerValueModifier/Presets/The Escape.json (hunk 2 above)
Comment (devilflippy) [reply user/mods/ServerValueModifier/Presets/The Escape.json]: as I disabled it, I didn't look at this numbers

File: user/mods/ServerValueModifier/Presets/The Escape.json (hunk 1 above)
Comment (devilflippy) [reply user/mods/ServerValueModifier/Presets/The Escape.json:126]: From my experience, it's better to keep it enabled even with realism

File: user/mods/ServerValueModifier/Presets/The Escape.json (hunk 2 above)
Comment (devilflippy) [reply user/mods/ServerValueModifier/Presets/The Escape.json]: fixed

**Review by guillaumearm — APPROVED** [review]

**Pull request merged**